SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -198,33 +200,72 @@ private Task ProcessPendingErrorsAsync()
             });
         }
 
-        private Task SendCrashReportsOrAwaitUserConfirmationAsync()
+        private async Task SendCrashReportsOrAwaitUserConfirmationAsync()
         {
-            return HandleUserConfirmationAsync();
+            bool alwaysSend = ApplicationSettings.GetValue(PrefKeyAlwaysSend, false);
+            if (_unprocessedManagedErrorLogs.Count() > 0)
+            {
+                // Check for always send: this bypasses user confirmation callback.
+                if (alwaysSend)
+                {
+                    AppCenterLog.Debug(LogTag, "The flag for user confirmation is set to AlwaysSend, will send logs.");
+                    await HandleUserConfirmationAsync(UserConfirmation.Send);
+                    return;
+                }
+
+                var shouldAwaitUserConfirmation = ShouldAwaitUserConfirmation?.Invoke();
+                if (shouldAwaitUserConfirmation == true)
```

Comment:
Checking a nullable boolean for `true` is technically correct but IMO it's not as readable as:

> shouldAwaitUserConfirmation.HasValue && shouldAwaitUserConfirmation.Value

Reply:
```suggestion
                if (shouldAwaitUserConfirmation.HasValue && shouldAwaitUserConfirmation.Value)
```